bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/TriggerGCService.java

```diff
@@ -71,7 +71,15 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
                 Map<String, Object> configMap = JsonUtil.fromJson(requestBody, HashMap.class);
                 Boolean forceMajor = (Boolean) configMap.getOrDefault("forceMajor", null);
                 Boolean forceMinor = (Boolean) configMap.getOrDefault("forceMinor", null);
-                bookieServer.getBookie().getLedgerStorage().forceGC(forceMajor, forceMinor);
+                Double majorCompactionThreshold = (Double) configMap.getOrDefault("majorCompactionThreshold", null);
+                Double minorCompactionThreshold = (Double) configMap.getOrDefault("minorCompactionThreshold", null);
+                Long majorCompactionMaxTimeMillis = Long.valueOf((Integer) configMap.getOrDefault(
```

Comment:
use `(Long) configMap.getOrDefault(
                        "majorCompactionMaxTimeMillis", null)`?

Reply:
The value passed here is Integer by default.